doc/radosgw/account.rst

```diff
@@ -252,3 +287,5 @@ This example uses `awscli`_ to create an IAM user for S3 operations.
 .. _Evaluating policies within a single account: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_evaluation-logic.html#policy-eval-basics
 .. _Cross-account policy evaluation logic: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_evaluation-logic-cross-account.html
 .. _awscli: https://docs.aws.amazon.com/cli/latest/
+.. _Bucket Notifications: ../notifications/
+.. _Supported Zone Features: ../zone-features/supported-features
```

Comment:
this one is a broken link (https://ceph--59390.org.readthedocs.build/en/59390/radosgw/zone-features/supported-features returns 404)

i wonder if `../zone-features/#supported-features` would work? if not i think it would be fine to link to `../zone-features` since "supported features" is near the top

Reply:
`#` did the trick. The link seems to be working now.